atlasdb-dbkvs-hikari/src/main/java/com/palantir/nexus/db/pool/config/OracleConnectionConfig.java

```diff
@@ -103,6 +108,9 @@ public ConnectionProtocol getProtocol() {
     @Value.Auxiliary
     public Properties getHikariProperties() {
         Properties props = new Properties();
+        if (getConnectionParameters().isPresent()) {
+            props.putAll(getConnectionParameters().get());
```

Comment:
nit: the optional `isPresent() & get()` block might be better replaced as `props.putAll(getConnectionParameters().or(ImmutableMap.of()));`